__tests__/app.test.js

```diff
@@ -66,6 +66,7 @@ describe('testing GET /api/articles/:article_id', () => {
             votes: 0,
             article_img_url:
               "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: 0
```

Comment:
This should be a new test - rather than an amendment of an old one. Please create a new test :)